netty/src/main/java/io/grpc/netty/NettyServerHandler.java

```diff
@@ -320,7 +321,6 @@ private void sendResponseHeaders(ChannelHandlerContext ctx, SendResponseHeadersC
 
   private void cancelStream(ChannelHandlerContext ctx, CancelServerStreamCommand cmd,
       ChannelPromise promise) {
-    cmd.stream().abortStream(cmd.reason(), false);
```

Comment:
Yup ... restored.

I'm don't recall whether this class ever listened for streamInactive. It looks like we manually handle all RST_STREAM cases.